src/prerender.ts

```diff
@@ -168,7 +168,9 @@ export async function prerender(nitro: Nitro) {
         _route.data = new TextEncoder().encode(value);
       },
     });
-    if (res.status !== 200) {
+    // https://developer.mozilla.org/en-US/docs/Web/HTTP/Redirections
+    const redirectCodes = [301, 302, 303, 304, 307, 308];
+    if (![200, ...redirectCodes].includes(res.status)) {
```

Comment:
Maybe as a feature we can try to follow redirects by checking `Location` header?

Reply:
That would be nice yes. I think we can merge this and do it in a separated PR.

Reply:
If not too much trouble introducing in same would be nice (so final changelog wouln't be a fix+feat)

Reply:
/cc @Hebilicious are you still on this? Please let me know if needed help

Reply:
I should have time to work on this in the upcoming days